LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
could you change this to "2017 Chris Johnson"?

Reply:
Sure! I'll do that right now.